README_BN.md

```
@@ -0,0 +1,384 @@
+-e PROXY_URL="http://127.0.0.1:7890 user pass"
+```
+
+### Shell
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/KhulnaSoft/NextGPT/main/scripts/setup.sh)
+```
+
+## ডকুমেন্টেশন
+
+> আরও ডকুমেন্টেশন নির্দেশনার জন্য [docs][./docs] ডিরেক্টরিতে যান।
```

Markdownlint (0.37.0)

Missing link or image reference definition: "./docs"
Reference links and images should use a label that is defined

(MD052, reference-links-images)